src/main/java/io/beanmapper/autoconfigure/BeanMapperAutoConfig.java

```diff
@@ -52,11 +54,13 @@
     private ApplicationContext applicationContext;
     @Autowired(required = false)
     private BeanMapperBuilderCustomizer builderCustomizer;
-    private ApplicationScanner appScanner;
+    private ApplicationScanner collectionHandlerAppScanner;
+    private ApplicationScanner beanConverterAppScanner;
 
     @PostConstruct
     private void initApplicationScanner() {
-        appScanner = new ApplicationScanner(applicationContext);
+        collectionHandlerAppScanner = new ApplicationScanner(applicationContext);
+        beanConverterAppScanner = new ApplicationScanner(applicationContext);
```

Comment:
Waarom 2 verschillende instanties nodig?

Reply:
Both classes maintained their own lists. If the scanner is reused, the BeanConverter list also contains the residue of the scanned collection handler.